store/src/java/com/zimbra/cs/zimlet/ZimletUtil.java

```diff
@@ -195,24 +195,48 @@ public static ZimletPresence getAvailableZimlets(Account acct) throws ServiceExc
         if (domain != null) {
             String[] domainZimlets = domain.getMultiAttr(Provisioning.A_zimbraZimletDomainAvailableZimlets);
             for (String zimletWithPrefix : domainZimlets) {
-                availZimlets.put(zimletWithPrefix);
+                if(isNotZextrasZimlet(zimletWithPrefix)) {
+                    availZimlets.put(zimletWithPrefix);
+                }
             }
         }
 
         String[] acctCosZimlets = acct.getMultiAttr(Provisioning.A_zimbraZimletAvailableZimlets);
         for (String zimletWithPrefix : acctCosZimlets) {
-            availZimlets.put(zimletWithPrefix);
+            if(isNotZextrasZimlet(zimletWithPrefix)) {
+                availZimlets.put(zimletWithPrefix);
+            }
         }
 
         return availZimlets;
     }
+    
+    private static boolean isNotZextrasZimlet(String zimletName) {
```

Comment:
could you  add some doc here for understanding what this method is actually doing with some example